### PR TITLE
fix: big number size in Slack and E-mail

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #5588 

### Description:
before
Screenshot 2023-06-07 at 17 02 42

after
Screenshot 2023-06-07 at 17 02 54
Screenshot 2023-06-07 at 16 48 07

### CODE REVIEW

File: packages/backend/src/services/UnfurlService/UnfurlService.ts [hunk 1]
```diff
@@ -130,7 +278,10 @@ export class UnfurlService {
 
             await page.setExtraHTTPHeaders({ cookie });
 
-            await page.setViewport(viewport);
+            // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+            chartType === ChartType.BIG_NUMBER
```
Comment (rephus) [inline packages/backend/src/services/UnfurlService/UnfurlService.ts:282]: why not doing if/else here ? that way you can avoid the eslint error

File: packages/backend/src/services/UnfurlService/UnfurlService.ts [hunk 2]
```diff
@@ -145,7 +296,7 @@ export class UnfurlService {
             });
             try {
                 await page.goto(url, {
-                    timeout: 150000, // Wait 2.5 mins for the page to load
+                    timeout: 300000, // Wait 2.5 mins for the page to load
```
Comment (rephus) [inline packages/backend/src/services/UnfurlService/UnfurlService.ts:299]: why did you increase this ?

File: packages/backend/src/services/UnfurlService/UnfurlService.ts [hunk 3]
```diff
@@ -155,7 +306,7 @@ export class UnfurlService {
             }
 
             const path = `/tmp/${imageId}.png`;
-
+            console.log(lightdashPage);
```
Comment (rephus) [inline packages/backend/src/services/UnfurlService/UnfurlService.ts:309]: leftover log

File: packages/backend/src/services/UnfurlService/UnfurlService.ts [hunk 4]
```diff
@@ -101,11 +107,153 @@ export class UnfurlService {
         this.projectModel = projectModel;
     }
 
+    async getTitleAndDescription(
```
Comment (rephus) [inline packages/backend/src/services/UnfurlService/UnfurlService.ts:110]: not sure why these methods have moved through the file, so I can't see if there have been any changes.

**Review by rephus — COMMENTED** [review]
Implementation looks good, just remove the log and revert the timeout.

File: packages/backend/src/services/UnfurlService/UnfurlService.ts (hunk 2 above)
Comment (yasmin-e) [reply packages/backend/src/services/UnfurlService/UnfurlService.ts:299]: oh my bad this was just debugging, i'll revert it

File: packages/backend/src/services/UnfurlService/UnfurlService.ts (hunk 4 above)
Comment (yasmin-e) [reply packages/backend/src/services/UnfurlService/UnfurlService.ts:110]: no changes but ESLint moved them on save 🤔

File: packages/backend/src/services/UnfurlService/UnfurlService.ts [hunk 5]
```diff
@@ -130,6 +278,10 @@ export class UnfurlService {
 
             await page.setExtraHTTPHeaders({ cookie });
 
+            if (chartType === ChartType.BIG_NUMBER) {
+                await page.setViewport(bigNumberViewport);
+            }
+
```
Comment (rephus) [inline packages/backend/src/services/UnfurlService/UnfurlService.ts:284]: if you don't do `else await page.setViewport(viewport);` you are going to be overwritting this bignumberviewport with the default

**Review by rephus — CHANGES_REQUESTED** [review]
just 1 more change

**Review by rephus — APPROVED** [review]

**Pull request merged**